src/ontology/mondo-ingest.Makefile

```diff
@@ -370,6 +370,9 @@ $(TMPDIR)/mondo.sssom.tsv: $(TMPDIR)/mondo_repo_built
 $(TMPDIR)/mondo.owl: $(TMPDIR)/mondo_repo_built
 	cp $(TMPDIR)/mondo/src/ontology/mondo.owl $@
 
+$(TMPDIR)/mondo-edit.owl: $(TMPDIR)/mondo_repo_built
```

Comment:
## Intelligently refresh if `mondo` has new commits?
That's what we're doing for `tmp/mondo.owl` and `tmp/mondo.sssom.tsv`. See:
- https://github.com/monarch-initiative/mondo-ingest/pull/551